Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -164,6 +164,54 @@ theorem range_euclideanQuadrant (n : ℕ) :
     (range fun x : EuclideanQuadrant n => x.val) = { y | ∀ i : Fin n, 0 ≤ y i } :=
   Subtype.range_val
 
+open ENNReal in
+theorem interior_quadrant (n : ℕ) (p : ℝ≥0∞) (a : ℝ) :
+    interior { y : PiLp p (fun _ : Fin n ↦ ℝ) | ∀ i : Fin n, a ≤ y i } =
+      { y | ∀ i : Fin n, a < y i } := by
+  let f : Fin n → (Π _ : Fin n, ℝ) →L[ℝ] ℝ := fun i ↦ ContinuousLinearMap.proj i
```

Comment:
- Have a look at `interior_pi_set`.
- Please add a lemma about `interior (Set.Ici f)`, where `f : ∀ i, X i`, with `univ.pi fun i => Ioi (f i)` in the RHS.
- I'm a bit worried about defeq abuse around `PiLp`, but this seems to happen all over the place.

Reply:
One more question: why not ask for the whole model to be convex? Then you can use `Convex.interior` to get convexity of the interior.

Reply:
@fpvandoorn and I were discussing this last week: indeed, convexity of the full range versus interior came up.

We are wondering if a larger change is needed, though --- and will post about this on zulip soon.